im.fluffychat.Fluffychat.json

```diff
@@ -0,0 +1,66 @@
+{
+    "app-id": "im.fluffychat.Fluffychat",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.38",
+    "sdk": "org.gnome.Sdk",
+    "command": "fluffychat",
+    "copy-icon": true,
+    "separate-locales": false,
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=pulseaudio",
+        "--share=network",
+        "--device=all",
+        "--filesystem=xdg-download",
+        "--talk-name=org.freedesktop.Notifications",
+        "--own-name=chat.fluffy.fluffychat"
```

Comment:
If it's uses this app-id, why not use it instead of `im.fluffychat.Fluffychat`? Anyway, such question should be asked to the upstream developers

Reply:
The issue is that this was a former domain and app-id for this app, but the domain isn't in control of upstream developers anymore. It can't be changed on iOS and Android tho, so internally, it has to stay the same. Current app domain is https://fluffychat.im/

Reply:
That's awkward, especially with renaming stuff etc.

Reply:
It is. The mainstream matrix client went through two renames (vector -> riot -> element) and when you look at the app id: 

https://play.google.com/store/apps/details?id=im.vector.app&hl=en&gl=US

Reply:
I didn't leave a comment here as I prefer "up to date" app-id rather than one that is technically correct but not pointing to actual working domain. (tldr no complaints from me)

Reply:
That was my thought process. This one is way more visible and might lead to people looking at a wrong domain.

I will also consult a possibility of building the linux build with a different app id from mobile versions if it's possible, so the ids might align in future.